### PR TITLE
Single Variable Diag View

### PR DESCRIPTION
**Pull request by esheehan-gsl**
Implement a single-variable view for diagnosticts. Showing a single variable is both easier to build and maintain compared to a dashboard, and gives more space to the charts so that they can be more easily read.

### CODE REVIEW

**Review by ian-noaa — COMMENTED** [review]
This looks pretty nice. Outside of my naming question - is clicking & dragging to select a region of the map expected to work to filter the histogram?

File: services/ui/src/js/components/ChartMap/ChartMap.js [hunk 1]
```diff
@@ -11,64 +11,46 @@ import {
...
+
+  static getObservedAttributes() {
+    return ["loop", "value-property"];
+  }
 
   constructor() {
     super();
 
     const shadowRoot = this.attachShadow({ mode: "open" });
     shadowRoot.innerHTML = VectorMapDiag.#STYLE + VectorMapDiag.#TEMPLATE;
```
[the remark this answers is not in the document]
Comment (esheehan-gsl) [reply services/ui/src/js/components/ChartMap/ChartMap.js]: Oops… good catch.

**Comment by esheehan-gsl** [comment]
> This looks pretty nice. Outside of my naming question - is clicking & dragging to select a region of the map expected to work to filter the histogram?

🤔 Good point. I was originally planning on doing this later, but I think you’re right: this should be done as part of this ticket. I should be able to get this done today.

**Comment by esheehan-gsl** [comment]
@ian-noaa, the selection is implemented now. It syncs between both maps, so I think we’re back to where we were before this rewrite in terms of functionality.

**Review by ian-noaa — APPROVED** [review]
Looks good! The trick on an ARM Mac was to run the API service in a container and the UI service locally. There might be a more elegant way of doing that but for the future, I'll just avoid the hassle by running on a Linux x86 box.

**Pull request merged**